packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -19,6 +19,8 @@ import {
   FEATURES,
   isLoose,
 } from "./features";
+import { isRequired } from "@babel/helper-compilation-targets";
+import compatData from "../data/compat.json";
```

Comment:
Please don't directly import `.json` files, since it won't work when moving to native ESM :pray:

Ref https://github.com/babel/babel/pull/12759

Reply:
💔 https://nodejs.org/api/esm.html#esm_json_modules is still experimental.